install.ps1

```diff
@@ -75,6 +75,41 @@ param (
 )
 $ErrorActionPreference = "Stop"
 
+# Function to test the network stack. Ping/GET requests to the resource to ensure that network stack looks good for installation
+function Test-WebConnection {
+    param (
+        [string]$url
+    )
+
+    Write-Host "[+] Checking for Internet connectivity ($url)..."
+
+    if (-not (Test-Connection $url -Quiet)) {
+        Write-Host "`t[!] It looks like you cannot access $url. Check your network settings." -ForegroundColor Red
+        Start-Sleep 3
+        exit 1
+    }
+
+    Write-Host "`t[+] Ping check for $url passed" -ForegroundColor Green
```

Comment:
I would remove this line because the internet check is getting too noisy. I think one message saying the check succeeded at the end of the function is enough.